Source/WebCore/crypto/openssl/CryptoAlgorithmAESCFBOpenSSL.cpp

```diff
@@ -63,11 +65,11 @@ static std::optional<Vector<uint8_t>> cfb8(const Vector<uint8_t>& key, const Vec
         // "output" in the encryption mode and "input" in the decryption mode.
         shiftRegister[AES_BLOCK_SIZE + (shift++)] = encrypt ? output[i] : input[i];
         if (shift == AES_BLOCK_SIZE) {
-            memcpy(shiftRegister, shiftRegister + AES_BLOCK_SIZE, AES_BLOCK_SIZE);
+            memcpySpan(shiftRegisterSpan, shiftRegisterSpan.subspan(AES_BLOCK_SIZE));
             shift = 0;
         }
     }
-    memset(encryptedBlock, 0, sizeof encryptedBlock);
+    zeroSpan(std::span { encryptedBlock });
```

Comment:
Are you sure you need the explicit `std::span` here? Implicit conversion should work.